src/usdb_syncer/gui/comment_dialog.py

```diff
@@ -0,0 +1,39 @@
+"""Dialog to post a comment on one or multiple songs."""
+
+from PySide6.QtWidgets import QDialog, QWidget
+
+from usdb_syncer.gui.forms.CommentDialog import Ui_Dialog
+from usdb_syncer.usdb_scraper import RequestMethod, get_usdb_page
+from usdb_syncer.usdb_song import UsdbSong
+
+
+class CommentDialog(Ui_Dialog, QDialog):
+    """Dialog to post a comment on one or multiple songs."""
+
+    def __init__(self, parent: QWidget, selected_song: UsdbSong) -> None:
+        super().__init__(parent=parent)
+        self._parent = parent
+        self.setupUi(self)
+        self._selected_song = selected_song
+        self.combobox_rating.addItem("neutral")
+        self.combobox_rating.addItem("positiv")
+        self.combobox_rating.addItem("negativ")
```

Comment:
Could we make that English labels, or icons?

Reply:
Just add another argument to `addItem()`. The second one will be retrievable with `currentData()`.

Reply:
Added the labels (as text)